crates/next-core/Cargo.toml

```diff
@@ -17,6 +17,7 @@ mime = "0.3.16"
 once_cell = "1.13.0"
 qstring = "0.7.2"
 regex = "1.6.0"
+reqwest = { workspace = true, features = ["stream"] }
```

Comment:
Is this still needed?

Reply:
Removed in a follow up. This was part of the initial work where we would make an actual HTTP request to a router server spun up by Next.js during init.